### DISCUSSION TITLE
fetch-pack: unexpected disconnect while reading sideband packet fatal: early EOF fatal: fetch-pack: invalid index-pack output

### DISCUSSION THREAD

**Discussion by BrianZimbe** [opencrvs/opencrvs-core#4283] [Technical Q&A] [answered]
fetch-pack: unexpected disconnect while reading sideband packet fatal: early EOF fatal: fetch-pack: invalid index-pack output

**Comment by euanmillar** [comment, marked as answer]
Hi @BrianZimbe 

This is not a problem with OpenCRVS, but with your network and Git installation.

Please try Google searching and you will find solutions, such as:

https://stackoverflow.com/questions/21277806/fatal-early-eof-fatal-index-pack-failed
https://confluence.atlassian.com/bitbucketserverkb/git-clone-fails-fatal-the-remote-end-hung-up-unexpectedly-fatal-early-eof-fatal-index-pack-failed-779171803.html
